hbase-common/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileContext.java

```diff
@@ -63,6 +63,8 @@
   private Encryption.Context cryptoContext = Encryption.Context.NONE;
   private long fileCreateTime;
   private String hfileName;
+  private byte[] columnFamily;
+  private byte[] tableName;
```

Comment:
Nit: I assume its byte[] instead of TableName here because that's what is available in the context. Otherwise TableName offers additional semantics that might be useful in the future, since we're passing it through now.

Reply:
Correct, byte[] was what was already there. I could have constructed a TableName and passed that through, but it seemed like an unnecessary allocation.